src/components/controls/TextInput/TextInput.scss

```diff
@@ -129,6 +133,10 @@ $block: '.#{variables.$ns}text-input';
                 padding-right: 1px;
             }
 
+            #{$block}__error-icon {
```

Comment:
Let's use inner variable smth like `--_--text-input-error-icon-padding` instead of this styles